Sources/MessagesViewController.swift

```diff
@@ -63,17 +61,10 @@ open class MessagesViewController: UIViewController {
         setupDelegates()
 
     }
-
+    
     open override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
-        if isFirstLayout {
-            setupMessageInputBarCopy()
-        }
-    }
-
-    open override func viewDidAppear(_ animated: Bool) {
-        super.viewDidAppear(animated)
-        removeMessageInputBarCopy()
+        messageInputBar.inputTextView.setNeedsDisplay()
```

Comment:
Was `setNeedsDisplay` left intentionally?

Reply:
oh no that was from testing, shouldn't be there